common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -383,3 +406,37 @@ void multi_instruction_init(const char **arr, const uint8_t count, const uint16_
 
     multi_instruction_create();
 }
+
+void multi_instruction_with_image_init(instruction_content_t content[], const uint8_t count, const uint16_t delay_in_ms, const bool destruct_on_click)
```

Comment:
ensure input parameters are not NULL and are within expected limits, eg: `count` must be less than `MAX_NUM_OF_INSTRUCTIONS`

Reply:
fixed